vpr/src/place/median_move_generator.cpp

```diff
@@ -88,53 +88,39 @@ e_create_move MedianMoveGenerator::propose_move(t_pl_blocks_to_be_moved& blocks_
             }
 
             const auto& net_bb_coords = cube_bb ? place_move_ctx.bb_coords[net_id] : union_bb;
-            //use the incremental update of the bb
-            ClusterBlockId bnum = cluster_ctx.clb_nlist.pin_block(pin_id);
-            int pnum = blk_loc_registry.tile_pin_index(pin_id);
-            VTR_ASSERT(pnum >= 0);
-            t_pl_loc block_loc = block_locs[bnum].loc;
-            t_physical_tile_type_ptr block_physical_type = physical_tile_type(block_loc);
-            int xold = block_loc.x + block_physical_type->pin_width_offset[pnum];
-            int yold = block_loc.y + block_physical_type->pin_height_offset[pnum];
-            int layer_old = block_loc.layer;
-
-            xold = std::max(std::min(xold, (int)device_ctx.grid.width() - 2), 1);  //-2 for no perim channels
-            yold = std::max(std::min(yold, (int)device_ctx.grid.height() - 2), 1); //-2 for no perim channels
-            layer_old = std::max(std::min(layer_old, (int)device_ctx.grid.get_num_layers() - 1), 0);
```

Comment:
Calls to `std::max()` and `std::min()` are removed from median move generators. There are a few other palces in this file and `weighted_median_move_generator.cpp` where pin location coordinates are clipped between 1 and H|W-2. QoR changes before and after the commit that removes this.